vision/google/cloud/vision/_gax.py

```diff
@@ -14,12 +14,36 @@
 
 """GAX Client for interacting with the Google Cloud Vision API."""
 
+from google.cloud.gapic.vision.v1 import image_annotator_client
+from google.cloud.grpc.vision.v1 import image_annotator_pb2
+
+from google.cloud._helpers import _to_bytes
+
 
 class _GAPICVisionAPI(object):
     """Vision API for interacting with the gRPC version of Vision.
 
-    :type client: :class:`~google.cloud.core.client.Client`
+    :type client: :class:`~google.cloud.vision.client.Client`
     :param client: Instance of ``Client`` object.
     """
     def __init__(self, client=None):
-        raise NotImplementedError
+        self._client = client
+        self._api = image_annotator_client.ImageAnnotatorClient
+
+
+def _to_gapic_feature(feature):
+    """Helper function to convert a ``Feature`` to a gRPC ``Feature``."""
+    feature_pb = image_annotator_pb2.Feature
+    feature_type = getattr(feature_pb, feature.feature_type)
+    return feature_pb(type=feature_type, max_results=feature.max_results)
+
+
+def _to_gapic_image(image):
+    """Helper function to convert an ``Image`` to a gRPC ``Image``."""
+    image_pb = image_annotator_pb2.Image
+    if image.content is not None:
+        return image_pb(content=_to_bytes(image.content))
+    if image.source is not None:
+        image_source_pb = image_annotator_pb2.ImageSource
+        image_source = image_source_pb(gcs_image_uri=image.source)
+        return image_pb(source=image_source)
```

Comment:
Right now, it returns `None`. But maybe it should throw an exception? Maybe a `ValueError`?

If neither `source` or `content` to be populated, that would mean they are either not passing  in an instance of `Image`. Since to instantiate `Image` requires you to populate one of those.